pkgs/applications/editors/neovim/utils.nix

```diff
@@ -162,22 +154,25 @@ let
     , extraName ? ""
   }:
     let
-      /* for compatibility with passing extraPythonPackages as a list; added 2018-07-11 */
-      compatFun = funOrList: (if builtins.isList funOrList then
-        (_: lib.warn "passing a list as extraPythonPackages to the neovim wrapper is deprecated, pass a function as to python.withPackages instead" funOrList)
-      else funOrList);
+
+      # we convert from the old configure.format to
+      plugins = if builtins.hasAttr "plug" configure then
+          throw "The neovim legacy wrapper doesn't support configure.plug anymore, please setup your plugins via 'configure.packages' instead"
```

Comment:
- [ ] This breaking change is missing a release note update.

Reply:
thanks for the comments, I will deal with them in a follow up PR.

Reply:
made the requested changes in https://github.com/NixOS/nixpkgs/pull/126633